po/uh/unknown-horizons.pot

```diff
@@ -1,222 +1,232 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR The Unknown Horizons Team
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# 
+#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: Unknown Horizons b829058\n"
+"Project-Id-Version: Unknown Horizons 5a638d3\n"
 "Report-Msgid-Bugs-To: team@unknown-horizons.org\n"
-"POT-Creation-Date: 2016-08-14 02:13+0200\n"
+"POT-Creation-Date: 2016-08-18 19:43+0200\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language: \n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
-#. (itstool) path: Container/Label@text
-#: content/gui/xml/editor/editor_settings.xml:6
-msgid "Select brush size:"
+#: run_uh.py:124
+msgid "Unknown Horizons has crashed."
 msgstr ""
 
-#. (itstool) path: Container/Label@text
-#: content/gui/xml/editor/editor_settings.xml:20
-msgid "Select terrain:"
+#: run_uh.py:126
+msgid "We are very sorry for this and want to fix the underlying error."
 msgstr ""
 
-#. (itstool) path: Container/Label@text
-#: content/gui/xml/editor/editor_settings.xml:27
-msgid "(right click to stop)"
+#: run_uh.py:127
+msgid "In order to do this, we need the information from the logfile:"
+msgstr ""
+
+#: run_uh.py:129
+msgid ""
+"Please give it to us via IRC or our forum, for both see http://unknown-"
+"horizons.org ."
+msgstr ""
+
+#: run_uh.py:178
+msgid "Error: Unable to find required library \"PyYAML\"."
+msgstr ""
+
+#: run_uh.py:179
+msgid "PyYAML (a required library) is missing and needs to be installed."
+msgstr ""
+
+#: run_uh.py:180
+msgid "The Windows installer is available at http://pyyaml.org/wiki/PyYAML."
+msgstr ""
+
+#: run_uh.py:181
+msgid ""
+"Linux users should find it using their package manager under the name "
+"\"pyyaml\" or \"python-yaml\"."
+msgstr ""
+
+#: run_uh.py:215
+msgid "Thank you for using Unknown Horizons!"
 msgstr ""
 
-#. (itstool) path: VBox/Label@text
 #: content/gui/xml/editor/editor_pause_menu.xml:14
 msgid "Return to editor"
 msgstr ""
 
-#. (itstool) path: VBox/Label@text
 #: content/gui/xml/editor/editor_pause_menu.xml:15
+#: horizons/gui/modules/select_savegame.py:53
 msgid "Save map"
 msgstr ""
 
-#. (itstool) path: VBox/Label@text
```

Comment:
It is OK, it is the same like the actual code. Maybe too much changes in last time. I got a marge error at last.

https://github.com/unknown-horizons/unknown-horizons/blob/master/po/uh/unknown-horizons.pot